packages/express-list-endpoints/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@prairielearn/express-list-endpoints",
+  "version": "1.0.0",
+  "license": "MIT",
+  "type": "module",
+  "main": "dist/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/PrairieLearn/PrairieLearn.git",
+    "directory": "packages/express-list-endpoints"
+  },
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc --watch --preserveWatchOutput",
+    "test": "c8 mocha src/**/*.test.ts"
+  },
+  "devDependencies": {
+    "@prairielearn/tsconfig": "workspace:^",
+    "@types/node": "^20.16.2",
+    "c8": "^10.1.2",
+    "chai": "^5.1.1",
+    "express": "^4.21.0",
+    "express-4-19-2": "npm:express@4.19.2",
```

Comment:
Do we actually care about testing with legacy versions of Express? We could avoid an extra dependency and the corresponding Dependabot security warnings if we're willing to say that we only support the most recent versions of Express. Note that this PR does not update PL/etc. to use the latest version of Express.

Reply:
I see no need to test with 4.19 or other older versions of express. In general, I think it's fine for us to say "we currently support only version X of express" (where X might not be the latest).

Reply:
In 52469ab, I removed the tests for older versions of Express. Note that because of Yarn deduping, we are in fact now using `express@4.21.0` in all our applications.